src/Revision/EntityRevisionLogTrait.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\entity\Revision\EntityRevisionLogTrait.
+ */
+
+namespace Drupal\entity\Revision;
+
+use Drupal\Core\Field\BaseFieldDefinition;
+
+/**
+ * Provides a trait implementing \Drupal\entity\Revision\EntityRevisionLogInterface.
+ */
+trait EntityRevisionLogTrait {
+
+  /**
+   * Provides the base fields for the entity revision log trait.
+   *
+   * @return \Drupal\Core\Field\BaseFieldDefinition[]
+   */
+  protected static function enhancedEntityRevisionDefaultBaseFields() {
```

Comment:
The name still needs work - it does not match the trait name -> entityRevisionLogBaseFieldDefinitions()